eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,49 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+function Get-AllPackageInfoFromRepo ($serviceDirectoryName="*")
+{
+  $allPackageProps = @()
+  Push-Location $RepoRoot
+  $allSetupProps = $null
+  try
+  {
+    pip install packaging==20.4 -q -I
+    $allSetupProps = (python -c "import sys; import os; sys.path.append(os.path.join('eng', 'scripts')); \
+    import get_package_properties; get_package_properties.get_all_package_properties('$serviceDirectoryName')")
+  }
+  catch
+  {
+    # This is soft error and failure is expected for python metapackages
+    LogError "Failed to get all package properties"
+  }
+  Pop-Location
+
+  foreach ($line in $allSetupProps)
+  {
+    $setupInfo = $line -Split ","
+    $packageName = $setupInfo[0].Trim("(' ")
+    $packageVersion = $setupInfo[1].Trim("' ")
+    $isNewSdk = $setupInfo[2].Trim()
+    $pkgDirectoryPath = Resolve-Path (Join-Path -Path $RepoRoot ($setupInfo[3].Trim(")' ")))
```

Comment:
Just for better documentation can you also capture `setupInfo[3]` into a variable that calls out what it is.